### ISSUE TITLE
Managing BigTable connections with BigTable client

### ISSUE THREAD

**Issue by clarencedesmarais** [googleapis/nodejs-bigtable#1190]
I am using BigTable in a typical NodeJs Javascript backend application. 

I'd like to get more clarity on how to setup BigTable in a slightly more complex microservice setup where we need to make requests through BigTable from different files which contain handler functions.

To put this simply, let's assume : 
```
src
|__handlers
  |__file1.js // needs to communicate with BigTable instance A for X operation
  |__file2.js // needs to communicate with BigTable instance A for Y operation
|__bigtable
   |__index.js // where bigtable communications are managed
|__index.js //boostrapping the server application here
```

Assuming we initialize bigtable via `const bigTable = new BigTable(options)` and then recuperate the instance with `bigTable.instance(...)` in bigtable/index.js. 

What would be the recommendation from Google if we need to call BigTable from multiple places (file1, file2) ? 
option A) Call `new BigTable()` once, when bootstrapping the service. 
  Basically we would opt for a singleton pattern in NodeJs where we export an object in `bigtable/index.js`.

option B) Calling `new BigTable()` whenever we need to make the communication. 

I am trying to understand the implications of each approach. 

Perhaps option A is desirable for simplicity (single place where initialization is done) and to minimize the number of connections.Maybe going for something like option B is totally acceptable too and might help us avoid bottlenecks due to the increased number of connections. 🤔  

Just hoping to get some guidance there as I am getting familiar with how this works generally :)

**Comment by danieljbruce** [comment]
Option A might be a lot better for latency since the first time the data client or admin client is used it needs to be initialized. If a client is shared across multiple files and the data client is initialized in one file because it is used there then it does not need to be initialized again in another file when it is used in another file so this saves time on the second use of the client.

**Comment by danieljbruce** [comment]
Closing this since the last comment answers the question. Please reopen if more advice is needed.
